versioned_docs/version-0.26/guides/getting-started/connect-to-your-cluster.md

```diff
@@ -0,0 +1,38 @@
+---
+id: connect-to-your-cluster
+title: Connect to your cluster
+---
+
+## Installation
+
+Install the appropriate package.
+
+```bash
+npm i -g zbctl
```

Comment:
We had a discussion internally whether this is the best installation source. I think the NPM package is maintained by Josh and not up to date all the time. On the plus side it is a very easy installation.
Our internal discussion didn't yield any clear preference, so it might be up for product management to decide whether they want to elevate the NPM package to officially supported or reduce its usage,